Scripts/modelsystem.py

```diff
@@ -185,6 +195,24 @@ def run(self, impedance, is_last_iteration=False):
                         for ass_class in impedance[tp][mtx_type]:
                             cost_data = impedance[tp][mtx_type][ass_class]
                             mtx[ass_class] = cost_data
+        # print impedance for convergence check
+        # (OBS! these o-d pairs only work for the test network;
+        #  the real o-d pairs are currently hard coded in another branch
+        #  for calibration, and should be implemented differently later)
+        car_times = {}
+        car_times["aht_od_1"] = impedance["aht"]["time"]["car_work"][
+            self.zdata_base.zone_index(5),self.zdata_base.zone_index(7)]
+        car_times["aht_od_2"] = impedance["aht"]["time"]["car_work"][
+            self.zdata_base.zone_index(7),self.zdata_base.zone_index(5)]
+        car_times["iht_od_3"] = impedance["iht"]["time"]["car_work"][
+            self.zdata_base.zone_index(6),self.zdata_base.zone_index(7)]
+        car_times["iht_od_4"] = impedance["iht"]["time"]["car_work"][
+            self.zdata_base.zone_index(7),self.zdata_base.zone_index(6)]
```

Comment:
You could make this a for loop, for instance looping over list of lists [["aht", 5, 7], ["aht", 7, 5]...]. The dict keys in car_times can be concatenated from these lists.

Reply:
That can be done quite easily I guess, but the problem is, where should the list be saved? And should we have some kind of "switch" in the config file to choose whether the test network or the real network is used?

Reply:
I think we could put the list in a file in the folder Scenario_input_data. Then no switch is needed, only zones that exist in the current scenario will be evaluated.

Reply:
Ok, I'll try to do that, and we'll see how it goes...